### ISSUE TITLE
Successful status disappears

### ISSUE THREAD

**Issue by m1dst** [Rafiuth/Soggfy#26]
Normally the successful check mark is displayed once a song is complete.  After a while, the check disappears and the tracks look like it they havent downloaded.  It also used to show a checkmark even after a spotify restart.  This doesn't seem to work either.  If you scroll up and down the playlist you can make them disappear too.

![image](https://user-images.githubusercontent.com/958968/161547742-a717e663-930d-4950-9fd4-c88748eb94a8.png)

Using latest release and latest support version of Spotify client.

Here are my settings....

![image](https://user-images.githubusercontent.com/958968/161549115-45729655-1c63-4b5f-ab8d-ab9269172886.png)

I was previously using v2.2.5 and OGG but I upgraded to see if another issue I was seeing was resolved.  I used to keep the files in ogg format however I found out over the weekend that some apps couldn't process the file, including ffmpeg.  I have reported that as a separate issue. #27 

Here is a screenshot after it has been running a while.  You will see a number of missing checkmarks even though the file has been downloaded.

![image](https://user-images.githubusercontent.com/958968/161552205-9429ea7f-38c7-4b27-aecf-2984b85f27bc.png)

**Comment by m1dst** [comment]
I wonder if it is due to paging virtualization and once you scroll about, that information is gone.  That would explain why they disappear during use.

**Comment by m1dst** [comment]
Jumping between playlist > queue > playlist wipes them out too.

**Comment by Rafiuth** [comment]
Could this be because you are using a network drive? Soggfy will query if the file exists everytime the page is refreshed (scrolled, reloaded, etc)

**Comment by Rafiuth** [comment]
I think this is because of the UNC prefix (\\), v2.2.8 added support for long paths and it always prefixes paths with `\\?\`.

**Comment by Rafiuth** [comment]
This should be fixed with cd115b0.

[Build here](https://github.com/Rafiuth/Soggfy/actions/runs/2093662664)

**Comment by m1dst** [comment]
I confirm this build fixes the problem. :-)
